optuna/distributions.py

```diff
@@ -233,15 +233,18 @@ class IntUniformDistribution(BaseDistribution):
             Upper endpoint of the range of the distribution. ``high`` is included in the range.
     """
 
-    def __init__(self, low, high):
-        # type: (int, int) -> None
+    def __init__(self, low, high, q):
```

Comment:
How about specifying a default value to keep backward compatibility.

```suggestion
    def __init__(self, low, high, q=1):
```